pars-derive/src/lib.rs

```diff
@@ -168,10 +161,10 @@ fn maybe_generate_from_str(ident: &syn::Ident, mode: &Mode) -> TokenStream {
 fn generate_from_str(ident: &syn::Ident) -> TokenStream {
     quote! {
         impl ::std::str::FromStr for #ident {
-            type Err = ::pars::Error;
+            type Err = pars::MatchError<'static>;
```

Comment:
I think we should use `::pars::` here for maximum namespace compatibility -- this would allow them to not have pars in scope and only have it in the source root.

Reply:
ah gotcha, that makes sense